.prettierrc

```diff
@@ -1,3 +1,7 @@
 {
-  "bracketSameLine": true
```

Comment:
all these configs has been taken from the best opensource projects, i would suggest to checkout this branch see all things working first-hand.

Reply:
> ...all these configs have been taken from the best opensource projects...

No doubt there but let's focus on developing the website's source code first rather than worry about configuring Prettier :laughing:. Relying on the [standard defaults Prettier provides](https://prettier.io/docs/en/options.html) is good enough for now.